Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -220,4 +240,21 @@ async function updateScmType(SPN, webAppName: string, resourceGroupName: string,
     }
 }
 
+function addWebConfigForNode(folderPath, webConfigParameters) {
```

Comment:
Is it better to have a generic function that takes app type and modifies the web.config file accordingly ?